libs/providers/launchdarkly-client/src/lib/launchdarkly-provider-options.ts

```diff
@@ -36,5 +36,5 @@ export interface LaunchDarklyProviderOptions {
    * If the provider should share a logger with the SDK, then you will need to
    * provide the logger in the SDK options as well.
    */
-  logger?: LDLogger  | Logger;
+  providerLogger?: LDLogger  | Logger;
```

Comment:
personally, I think this is clear enough and we can just keep "logger" but that's a nit.

Reply:
Agree!!